packages/next/src/views/Version/index.tsx

```diff
@@ -135,10 +118,10 @@ export const VersionView: EditViewComponent = async (props) => {
     <DefaultVersionView
       doc={doc}
       docPermissions={docPermissions}
-      initialComparisonDoc={mostRecentDoc}
+      initialComparisonDoc={latestVersion}
+      latestDraftVersion={latestDraftVersion.id}
```

Comment:
@JessChowdhury  when collection is set to version:true (without drafts) those two linese are causing error. 
![image](https://github.com/user-attachments/assets/8772b0da-fe67-4f7a-b0b7-9d4098653770)